packages/node_modules/@node-red/editor-client/locales/ko-KR/editor.json

```
@@ -0,0 +1,909 @@
+{
+  "common": {
+    "label": {
+      "name": "이름",
+      "ok": "확인",
+      "done": "완료",
+      "cancel": "취소",
+      "delete": "삭제",
+      "close": "닫기",
+      "load": "열기",
+      "save": "저장",
+      "import": "가져오기",
+      "export": "내보내기",
+      "back": "뒤로",
+      "next": "앞으로",
+      "clone": "프로젝트 복제",
+      "cont": "계속하기"
+    }
+  },
+  "workspace": {
+    "defaultName": "순서도 __number__",
```

```suggestion
    "defaultName": "플로우 __number__",
```
순서도 -> 플로우(peullou)
It would be better to use it as pronounced. Because it is a proper noun.